docs/source/architecture/period_duration.rst

```diff
@@ -0,0 +1,61 @@
+Period Duration
+===============
+
+In the NuCypher Network, a *Period* is the minimum unit for policy duration. This applies to sharing policy durations
+and also corresponds to the frequency that Workers must make an on-chain commitment to being online and available.
+Periods begin at UTC midnight.
+
+Genesis
+-------
+
+At the Network launch on October 15th, 2020, the duration of a period was 24 hours. As a result, before UTC midnight,
+Workers needed to make a per period (daily) on-chain commitment transactions indicating availability for the following
+period (after UTC midnight).
+
+The cost of the commitment transaction was approximately 200k gas but could cost more depending on the number of
+sub-stakes being staked. Given the significant increase in Ethereum gas since launch, the cost of the daily
+commitment transaction significantly increased and became untenable. Particularly for stake sizes closer to the minimum
+stake size, the NU staking rewards and fees received from running a Worker may not cover the cost of this commitment
+transaction.
+
+Reducing the transaction gas cost was a necessity, and there were multiple mitigation attempts:
```

Comment:
This is not exactly accurate since transactions still cost the same amount, there are just fewer of them required.

```suggestion
 Reducing the total amount of gas fees required to run a worker became a necessity, and there have been multiple mitigations:
```

Reply:
✔️